woocommerce/woocommerce-gutenberg-products-block
# Remove the fontSize for the button in the `Product Collection 4 Column` pattern

## What
Removes the font size setting for the `Add to Cart Button` block in the `Product Collection 4 Column` so the button looks better.

## Why
Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11403


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page or post.
2. Insert the `Product Collection 4 Column` pattern.
3. Check the `Add to cart` buttons look like in the after screenshot below.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|  <img width="1201" alt="Screenshot 2023-10-24 at 15 53 25" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/cc60d499-51b9-4a12-8f11-6c346eb41ce8"> |    <img width="1197" alt="Screenshot 2023-10-24 at 15 53 01" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/a1735c13-f701-46e4-bbe0-b2b2eff275bf">   |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Improve the button size on the "Product Collection 4 Column" pattern.
